File: chapter1/mario/mario2.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for the number of ?
    int q;
    do
    {
        q = get_int("Number: ");
    }
    while (q <= 0);

    // For each question mark
    for (int i = 0; i < q; i++)
    {
        printf("?");
    }

    // Print a new line
    printf("\n");
}

File: chapter5/mario2.py
from cs50 import get_int

def main():
    # Get height from the user
    n = get_height()

    # Print the blocks horizontally
    for i in range(n):
        print("#", end="")

    # New line
    print()

def get_height():
    while True:
        n = get_int("Height: ")
        if n > 0:
            break
    return n

main()
